plugin/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "kubectl-openebs"
+version = "0.1.0"
```

Comment:
Shouldn't this be 1.0.0 ? Thoughts @tiagolobocastro @avishnu

Reply:
hmm why should it be 1.0.0?

Reply:
because current openebsctl is 0.6.0, we don't want to go back in version right?

Reply:
And now that we are doing a major restructuring and rewrite it should be a major version bump. Also, openebsctl is in krew so we cannot publish older versions.

Reply:
btw a breaking change on 0.6.0 is not 1.0.0, but rather 0.7.0

Reply:
@avishnu any suggestions?

Reply:
It's not a new plugin right it's still `kubectl-openebs`? Yeah maybe `0.7.0` but `0.1.0` sounds wrong. Consider the fact that this will go on krew with same name.

Reply:
Oh doh I see, on kew it's openebs, not openebsctl -.-
So if we want to keep on krew then we'd need to be 0.7.0 indeed, unless we were to change the name

Reply:
Yeah. That was my major concern. I would say 0.7.0 since a breaking change is allowed here.

Reply:
I think 0.7.0 makes sense. But, what's the criteria for moving major from 0 to 1?

Reply:
Don't think there's a hard and fast rule, but usually 1 is used when things have been stable for a while (public api included), with perhaps no major changes foreseen in short term. 
Many times I've seen v1 being released under the guile of stability and feature completeness :)

Reply:
Lets keep it 0.7.0 ?

Reply:
Yep, 0.7.0 seems the right call here, allowing us to remain on krew